Library/Formula/speech-tools.rb

```diff
@@ -0,0 +1,23 @@
+class SpeechTools < Formula
+  homepage "http://festvox.org/docs/speech_tools-2.4.0/"
+  url "http://festvox.org/packed/festival/2.4/speech_tools-2.4-release.tar.gz"
+  sha1 "5b0ebd39bb7afa33e4093724a2123bdc62a6aebc"
+
+  def install
+    ENV.deparallelize
+    system "./configure"
+    system "make"
+    bin.install "main/ch_track"
+    bin.install "main/ch_wave"
+    bin.install "main/pda"
+    bin.install "main/pitchmark"
+    bin.install "main/sig2fv"
+    bin.install "main/sigfilter"
+    bin.install "main/spectgen"
+    bin.install "main/wagon"
+  end
+
+  test do
+    assert_equal "Error reading ESPS file -", `ch_track 2>&1`.strip
```

Comment:
A thought occurs: wouldn't it make more sense to hook the test block into the `test` makefile target? Something along the lines of
```ruby
  test do
    Dir.chdir(prefix) do
      system "make", "test"
    end
  end
```
(which incidentally actually enters the `testpath`, not the keg, directory, and therefore doesn't do the trick -- sorry, I'm new to this).
Or am I on the wrong track here?